app/views/people/show.mobile.haml

```diff
@@ -10,6 +10,8 @@
         = @person.name
       %span.description
         = @person.diaspora_handle
+      .useraspects
```

Comment:
I think at first I did not know what I wanted and I started with a div with a class. I have transformed it into a div, since the select itself has some id now and I don't need this anymore.